README.md

```diff
@@ -8,6 +8,13 @@ Written in rust, uses cargo. In linux you can install this with
 Uses z3-sys, which requires clang. To install this on linux, use
 `sudo apt-get install clang`.
 
+If you are using other linux program like ubuntu, and found error,
+cargo : The term 'cargo' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Comment:
Put `` around cargo : The term 'cargo' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.